test/generator/exercise_test.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../test_helper'
+
+module Generator
+  class ExerciseTest < Minitest::Test
+    def test_slug
+      exercise = Exercise.new(slug: 'alpha')
+      assert_equal 'alpha', exercise.slug
+    end
+
+    def test_name
+      exercise = Exercise.new(slug: 'alpha-beta')
+      assert_equal 'alpha_beta', exercise.name
```

Comment:
Yay for tests.
I'm not sure `name` is the right name for this property, but it is consistent with what we've been using.